tests/lib/rules/post-message-no-wildcard-targets.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @fileoverview Disallow using the wildcard `*` in postMessage
+ * @author Automattic
+ * @copyright 2017 Automattic. All rights reserved.
+ * See LICENSE.md file in root directory for full license.
+ */
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var rule = require( '../../../lib/rules/post-message-no-wildcard-targets' ),
+	config = { env: { es6: true } },
+	RuleTester = require( 'eslint' ).RuleTester;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+( new RuleTester( config ) ).run( 'post-message-no-wildcard-targets', rule, {
+	valid: [
+        { code: 'foo()' },
```

Comment:
Hmm, why aren't these failing on mixed spacing? Lint might not be covering tests.